vrecord

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env bash
+
+version="2016-11-19"
```

Comment:
No idea! A date is better than an empty variable…

But we can also delete the printing of this variable at https://github.com/amiaopensource/vrecord/blob/b04905def37a5476a144b139981bba96bf23e57a/vrecord#L24